ravendb/rvn_server.go

```diff
@@ -0,0 +1,760 @@
+	var stdoutBuf bytes.Buffer
+	var conn *ssh.Client
+	defer func() {
+		log.Println(stdoutBuf.String())
+	}()
+	ravenPackageUrl := "https://daily-builds.s3.us-east-1.amazonaws.com/ravendb_" + sc.Version + "-0_amd64.deb"
+
+	signer, err := ssh.ParsePrivateKey(sc.SSH.Pem)
+	if err != nil {
+		return err
+	}
+	authConfig := &ssh.ClientConfig{
+		User:            sc.SSH.User,
+		Auth:            []ssh.AuthMethod{ssh.PublicKeys(signer)},
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
+		Timeout:         time.Minute * 10,
+	}
+	for i := 0; i < numberOfIterationsToDial; i++ {
+		conn, err = ssh.Dial("tcp", net.JoinHostPort(publicIP, fmt.Sprint(sc.SSH.getPort())), authConfig)
+		if err == nil {
+			break
+		} else {
+			if i != numberOfIterationsToDial {
+				time.Sleep(time.Second * 5)
+			} else {
+				return err
+			}
+		}
+	}
+
+	defer conn.Close()
+	for i := 0; i < numberOfIterationsToDial; i++ {
+		err = sc.execute(publicIP, []string{
+			"n=0; while [ \"$n\" -lt 10 ] && [ ! -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting for cloud-init...'; n=$(( n + 1 )); sleep 1; done",
+			"wget -nv -O ravendb.deb " + ravenPackageUrl,
+			"sudo apt-get update -y",
+			"sudo apt-get install -y -f ./ravendb.deb",
+		}, stdoutBuf, conn)
+		if err == nil {
+			break
+		} else {
+			if i != numberOfIterationsToDial {
+				time.Sleep(time.Second)
+			} else {
+				return err
+			}
+		}
+	}
+
+	err = upload(conn, stdoutBuf, "/etc/ravendb/license.json", sc.License)
+	if err != nil {
+		return err
+	}
+
+	contents, err := readFileContents("/etc/ravendb/settings.json", stdoutBuf, conn)
+	if err != nil {
+		return err
+	}
+
+	var settings map[string]interface{}
+	err = json.Unmarshal(contents, &settings)
+	if err != nil {
+		stdoutBuf.WriteString("Failed to parse JSON\n")
+		stdoutBuf.Write(contents)
+		return err
+	}
+
+	if sc.ClusterCertificate != nil && sc.Insecure == false {
+		settings["Security.Certificate.Path"] = "/etc/ravendb/certificate.pfx"
+		err = upload(conn, stdoutBuf, "/etc/ravendb/certificate.pfx", sc.ClusterCertificate)
+		if err != nil {
+			return err
+		}
+	}
+
+	scheme := "https"
+	if sc.Insecure {
+		settings["Security.UnsecuredAccessAllowed"] = "PublicNetwork"
+		scheme = "http"
+	}
+	httpUrl, err := sc.setupUrls(index, scheme, settings)
+	if err != nil {
+		return err
+	}
+
+	settings["ServerUrl"] = scheme + "://0.0.0.0:" + strconv.Itoa(sc.Url.HttpPort)
+	settings["ServerUrl.Tcp"] = "tcp://0.0.0.0:" + strconv.Itoa(sc.Url.TcpPort)
+	settings["Setup.Mode"] = "None"
+	settings["License.Path"] = "/etc/ravendb/license.json"
+
+	for key, value := range sc.Settings {
+		settings[key] = value
+	}
+
+	jsonOut, err := json.MarshalIndent(settings, "", "\t")
+	if err != nil {
+		return err
+	}
+
+	err = upload(conn, stdoutBuf, "/etc/ravendb/settings.json", jsonOut)
+	if err != nil {
+		return err
+	}
+	err = sc.execute(publicIP, []string{
+		"sudo chown -R ravendb:ravendb /etc/ravendb/",
```

Comment:
You're overriding everything DEB package has set for you. Please review this file and either remove this line or adjust only specific files - e.g. license one.

https://github.com/ravendb/ravendb/blob/v5.2/scripts/linux/pkg/deb/assets/ravendb/debian/ravendb.postinst#L75